docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -46,7 +49,7 @@ To include more resources in the native executable, create a `resources-config.j
 The patterns are valid Java regexps.
 Here we include all the XML files and JSON files into the native executable.
 
-Now, let's tell the `native-image` executable about this configuration file by tweaking the `native-image` execution in your `pom.xml`:
+Now, let's tell the `native-image` executable about this configuration file by tweaking the `native-image` execution in your `pom.xml` when using maven:
```

Comment:
I wouldn't add `when using maven`. We're talking about the `pom.xml` so it's clear enough IMHO.